web/ui/react-app/src/pages/graph/ExpressionInput.test.tsx

```diff
@@ -264,7 +262,7 @@ describe('ExpressionInput', () => {
 
   it('renders an execute Button', () => {
     const addon = expressionInput.find(InputGroupAddon).filterWhere(addon => addon.prop('addonType') === 'append');
-    const button = addon.find(Button);
+    const button = addon.find(Button).find({ color: 'primary' });
```

Comment:
Would this work and be more clear/specific?

```suggestion
    const button = addon.find(Button).find('.execute-btn');
```

(and then omit the check for the class name on the next line)

Reply:
Sure, that sounds good to me. I will change this